### DISCUSSION TITLE
Non programmer collaboration

### DISCUSSION THREAD

**Discussion by barkalot** [community/community#23879] [New to GitHub]
<p>Hello,</p>
<p>I am still learning Python with some basic projects.  But I was wondering is there a way to create a ‘Project’, then have it centrally accessed by different people?  So they would be able to add files, create comments, work on design ideas, share links and documents.  All in one controlled place.  Git might me able to do all this, as I’m still learning that too I don’t know.</p>
<p>It might be that I create folders for each topic, the share this project with another GitHub user.  So they can add or delete files, comment, etc…</p>
<p>Idea’s<br>
Design<br>
Source<br>
Resources</p>
<p>Best regards,<br>
Dan</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>In principle, Git is meant for working together. If it’s right for your idea depends on <em>how</em> you want to work. In particular Git is good with plain text files (making it easy to see and merge different versions), not so great with binary data (including stuff like PDF or Word documents). So if you want to work on text documents with Git (definitely recommended), use plain text or plain text based markup languages like Markdown or LaTeX. :slightly_smiling_face:</p>
<p>One thing to note is that Git is decentralized. With a group you are (usually) not working “on the same document”, instead everyone is working on their own version, and Git keeps track of the different versions and lets you merge them (turn them into a unified document). That’s also the main reason Git doesn’t work as well with binary files: If Git can’t understand them, it can’t support merging, and you have to figure out yourself how to merge different versions.</p>
<p>Finally, it’s usually good to have one <em>repository</em> (not different folder) per project. What constitutes a project boundary is a bit of a debate sometimes, though. :wink:</p>
